packages/docgen/lib/get-type-annotation.js

```diff
@@ -426,7 +465,12 @@ function getParamTypeAnnotation( tag, declarationToken, paramIndex ) {
 	const isQualifiedName = tag.name.includes( '.' );
 
 	try {
+		if ( babelTypes.isAssignmentPattern( paramToken ) ) {
+			paramToken = paramToken.left;
```

Comment:
What does `paramToken.left` represent?

Reply:
`paramToken.left` represents the "regular" part of the assignment pattern. The assignment pattern is split along the `=` so:

```ts
function fn(foo: string = '') //...
```

Gets split like so:

|`left`|`right`|
|---|---|
|`foo: string`|`= ''`|

`left` could technically be any kind of pattern, whether an Identifier, a rest param, an array or object destructuring pattern:

```ts
function fn([foo]: string[] = []) //...
```

for example, `left` would be `[foo]: string[]`.